### DISCUSSION TITLE
Issue with Incorrect App Name in Push Notifications

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#686] [General Discussion]
We are currently encountering an issue with our Android app version 4.6 (APK) where the app name specified in the Sunbird.properties file is not reflecting correctly in push notifications. Despite declaring the app name properly in the configuration file, we consistently receive push notifications with the title "Diksha" instead of the expected app name.

We have thoroughly investigated the issue and confirmed that the app name is correctly specified in both the Sunbird.properties file and the local_notification_config.json file located in the buildConfig folder. Additionally, we have conducted a comprehensive search throughout our codebase for any references to "Diksha" but found no relevant occurrences.

Furthermore, we have meticulously debugged the notification process but have not identified any underlying issues.

To provide further clarity, I have attached a screenshot illustrating the discrepancy between the expected and actual push notification titles.
![1000055414 (1)](https://github.com/Sunbird-Ed/Community/assets/72926048/ecb26808-8672-4a93-8308-12c2f1657bed)

We kindly request assistance in resolving this issue promptly as it impacts the user experience and branding consistency of our application. Any insights or guidance on potential solutions would be greatly appreciated.

**Comment by Ajoymaity** [comment]
@siddhishinde0723 This message is coming from notification formConfig. Please update the bellow form config
        "request": {
        "type": "config",
        "subType": "notification",
        "action": "get",
        "component": "app" 
    }

CC: @swayangjit @vinukumar-vs

**Reply by siddhishinde0723** [reply]
@Ajoymaity where will i find this file.
I can only see local_notofocation_config.json in notification config file.

**Reply by Ajoymaity** [reply]
@siddhishinde0723 check in Postman

**Reply by siddhishinde0723** [reply]
@Ajoymaity For apk also the form config is same?

**Reply by Ajoymaity** [reply]
@siddhishinde0723 You can update the form config, change the form response and update it

**Reply by siddhishinde0723** [reply]
@Ajoymaity Got it thank you for the guidance

**Reply by siddhishinde0723** [reply]
@Ajoymaity is there any service which we need to restart after form config changes.
Even after changing the config we are still getting Diksha in the notification

**Comment by swayangjit** [comment]
@siddhishinde0723  Have you cleared your data and tried it ?

**Reply by siddhishinde0723** [reply]
@swayangjit Yes, I did try, and when I read that, my name is updated, but it does not appears on the notification.

**Reply by swayangjit** [reply]
@siddhishinde0723 Have you checked the form read API what its returning ?

**Reply by siddhishinde0723** [reply]
@swayangjit Yes i have checked the read it is giving the updated data.

**Reply by swayangjit** [reply]
@siddhishinde0723 Then in the app also you should get the updated data. Please do debug and let us know

**Reply by siddhishinde0723** [reply]
@swayangjit i did debugging part as well but still getting Diksha in the notification.
We are getting notification every Thursday.

**Reply by siddhishinde0723** [reply]
@swayangjit We are still having problems with it. Would you or anyone else be able to assist us with this?

**Reply by swayangjit** [reply]
@siddhishinde0723  Can you please let us know if we can connect over this issue ?

**Reply by siddhishinde0723** [reply]
@swayangjit This problem has been resolved. I appreciate the advice.
